src/main/java/springboot/demo/model/Product.java

```diff
@@ -0,0 +1,20 @@
+package springboot.demo.model;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@Builder
```

Comment:
do we need builder for model with one field?

Reply:
You are very attentive! The eagle eye)